### DISCUSSION TITLE
have a difficulty in building the model

### DISCUSSION THREAD

**Discussion by neperchh** [neperfepx/neper#699] [Q&A] [answered]
Dear teacher, I would like to build a model in which there are some extremely small grains at the grain boundary. Is it possible to achieve this? The desired effect is shown in the image below. I look forward to your response.BEST WISH.
![5308ab46b96ba46e4c390df6c213bf0](https://github.com/neperfepx/neper/assets/127730078/4589ec79-355e-4356-8546-0f022faae4b7)

**Comment by smallrain1996** [comment]
As far as I know, Neper doesn't have this feature, you can model directly in software like CAD.

**Reply by rquey** [reply, marked as answer]
Yes, to replicate this morphology, my best guess would be to generate the tessellation with Neper and then move to a CAD program to add the spheres.

Good luck.
